src/pages/LPA-page/LPAPage.tsx

```diff
@@ -26,7 +26,7 @@ export const LPAPage = ({ errors }: LPAPageProps): JSX.Element => {
   }, []);
 
   return (
-    <>
+    <div className="govuk-!-width-two-thirds">
```

Comment:
I've added 2/3 width class to the top level div to most form pages. I looked into adding it to our FormPageHOC but then I realised that not all for pages have 2/3 width (i.e. Export page) so I have added it to the individual pages where needed

Reply:
I have added the className to FormPageHoc and removed the FormPageHoc wrapper for Export page as it wasn't needed